src/sdam/topology.ts

```diff
@@ -511,6 +513,13 @@ export class Topology extends TypedEventEmitter<TopologyEvents> {
     this.removeListener(Topology.TOPOLOGY_DESCRIPTION_CHANGED, this.s.detectShardedTopology);
 
     stateTransition(this, STATE_CLOSED);
+    this.s.description = new TopologyDescription(TopologyType.Unknown, new Map());
```

Comment:
```suggestion
    this.s.description = new TopologyDescription(
        TopologyType.Unknown,
        new Map(),
        options.replicaSet,
        undefined,
        undefined,
        undefined,
        options
      );
```

Not sure if this is right, but feels like it would make sense to set it to unknown but pipe through the options that made the original starting `TopologyDescription`. Thoughts?

Reply:
I'm not against this and it wouldn't go against the spec, so I'm happy to implement this